### ISSUE TITLE
Have you merged WANN and NEAT?

### ISSUE THREAD

**Issue by wangfrombupt** [yuedajiong/super-brain-weight-agnostic-neural-networks#5]
Hi yuedajiong,
    Have you completed the merging of  WANN and NEAT? If yes, could you please submit the code to this private repository? Many thanks.

**Comment by yuedajiong** [comment]
doing ...

**Comment by wangfrombupt** [comment]
OK:)

**Comment by yuedajiong** [comment]
You can see that I submitted my MEAT implementation few hours ago, very simple, still powerful.

**Comment by wangfrombupt** [comment]
Yes, I can see that, very quick update! I also noticed that you have added some comments to the code. Can I also add comments to the code? I did so in the downloaded copy to assist my understanding of the code. But maintaining two versions of the code(the online version you update and the offline version I commented) would be a waste of time. And IMHO, my comments may increase the readability of your code:)

**Comment by yuedajiong** [comment]
Of course,  you can modify and submit.  

BUPT?  Chinese?  My wechat is john122333, I am in BeiJing.

**Comment by wangfrombupt** [comment]
Yeah, I am from BUPT. I will invite you in wechat. I am in Beijing too:)
